app/Http/Controllers/PostController.php

```diff
@@ -19,25 +19,27 @@ public function getAllPosts()
 
     public function createPost(Request $request)
     {
-        $user = Auth::user();
-        $title = $request->title;
-        $body = $request->body;
-        $images = $request->images;
+        DB::transaction(function () use ($request) {
+            $user = Auth::user();
```

Comment:
@SlyCreator Here You're using the transaction function from the `DB` class but you haven't imported the DB class in this file yet.
You can simply:
```php 
use DB;
```
before calling it in the `createPost` function